qalculate-qt-flatpak.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/data/qalculate-qt.appdata.xml b/data/qalculate-qt.appdata.xml
+index 4c2e95b..2085e45 100644
+--- a/data/qalculate-qt.appdata.xml
++++ b/data/qalculate-qt.appdata.xml
+@@ -63,9 +63,6 @@
+   <kudo>UserDocs</kudo>
+  </kudos>
+  <launchable type="desktop-id">qalculate-qt.desktop</launchable>
+- <provides>
+-  <binary>qalculate-qt</binary>
+- </provides>
```

Comment:
Why is this patch needed? I guess it was copied from the GTK UI app, but I'm not sure why it was there either.

Reply:
I found this in the log of the GTK GUI flatpak (from before my own involvement and before it was added to flathub):
```
Fix AppData file
    
The provides tag describes the public interfaces this program provides.
The binary tag is a child tag of the provides tag.
It provides the name of a binary installed into a location in PATH.
However, flatpak doesn't export any binaries on the host.
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-provides
```
I have a faint memory (possible wrong) of the tag causing an error when building the flatpak previously, but it is at least not the case anymore.

It would only make my life easier if the patch can safely be removed.

Reply:
I believe it's ok to have this tag if the metainfo file comes from upstream source and doesn't cause errors, but asked the question in https://matrix.to/#/#flatpak:matrix.org just in case.

Reply:
It is only informational and shouldn't be a problem.

Reply:
I've now removed the patch.